ci/check-crates.sh

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env bash
+
+# input:
+# 	env:
+#  	 	- CRATE_TOKEN
+#   	- COMMIT_RANGE
```

Comment:
This indentation is odd.

Reply:
oh yeah. thank you! seems I broke my editor setting. fixed https://github.com/solana-labs/solana/pull/29009/commits/405522fcd4309c00dc2effcd26c18066686414bb